### PR TITLE
docs: Fix duplicate message and format of 1.17.2 release note and sync it with related security update to zh

### PR DESCRIPTION
**Pull request by wilsonwu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md [hunk 1]
```diff
@@ -47,17 +47,16 @@ Crash when a redirect url without a state parameter is received in the OAuth fil
   ([Issue #42749](https://github.com/istio/istio/issues/42749))
 
 - **Fixed** a bug that would cause unexpected behavior when applying access logging configuration based on the direction of traffic. With this fix, access logging configuration for `CLIENT` or `SERVER` will not affect each other.
-  [Issue # 43371](https://github.com/istio/istio/issues/43371)
+  ([Issue # 43371](https://github.com/istio/istio/issues/43371))
```
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.2/index.md:50]:
```suggestion
  ([Issue #43371](https://github.com/istio/istio/issues/43371))
```

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md [hunk 2]
```diff
@@ -67,7 +66,7 @@ Crash when a redirect url without a state parameter is received in the OAuth fil
 
 - **Fixed** an issue causing VMs using auto-registration to ignore labels other than those defined in a `WorkloadGroup`.
   ([Issue #32210](https://github.com/istio/istio/issues/32210))
+  ([PR #44021](https://github.com/istio/istio/pull/44021))
```
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.2/index.md:69]: Point to original PR: https://github.com/istio/istio/pull/44012

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md [hunk 3]
```diff
@@ -67,7 +66,7 @@ Crash when a redirect url without a state parameter is received in the OAuth fil
 
 - **Fixed** an issue causing VMs using auto-registration to ignore labels other than those defined in a `WorkloadGroup`.
   ([Issue #32210](https://github.com/istio/istio/issues/32210))
```
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.2/index.md:68]: This issue does not seem related at all to this particular text. Do we need another bullet?

Comment (ericvn) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md:68]: Text in 32210 release note:
```
**Fixed** reconciliation logic in the validation webhook controller to rate-limit
  the retries in the loop. This should drastically reduce churn (and generated logs)
  in cases of misconfiguration.
```

File: content/zh/news/releases/1.17.x/announcing-1.17.2/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+该版本修复了于 4 月 4 日发布的 [ISTIO-SECURITY-2023-001](/zh/news/security/istio-security-2023-001)
+中阐述的安全漏洞。
+本发布说明描述了 Istio 1.17.1 和 Istio 1.17.2 之间的不同之处。
+
+{{< relnote >}}
+
+## 安全更新{#security-updates}
+
+- __[CVE-2023-27487](https://github.com/envoyproxy/envoy/security/advisories/GHSA-5375-pq35-hf2g)__:
+  (CVSS Score 8.2, High)：客户端可能会伪造标 `x-envoy-original-path` 头信息。
```
Comment (windsonsea) [inline content/zh/news/releases/1.17.x/announcing-1.17.2/index.md:19]:
```suggestion
  (CVSS Score 8.2, High)：客户端可能会伪造 `x-envoy-original-path` 标头信息。
```

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md (hunk 2 above)
Comment (wilsonwu) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md]: Done

File: content/zh/news/security/istio-security-2023-001/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+{{< security_bulletin >}}
+
+## CVE
+
+### Envoy CVEs{#envoy-cves}
+
+- __[CVE-2023-27487](https://github.com/envoyproxy/envoy/security/advisories/GHSA-5375-pq35-hf2g)__:
+  (CVSS Score 8.2, High)：客户端可能会伪造标 `x-envoy-original-path` 头信息。
```
Comment (windsonsea) [inline content/zh/news/security/istio-security-2023-001/index.md:21]:
```suggestion
  (CVSS Score 8.2, High)：客户端可能会伪造 `x-envoy-original-path` 标头信息。
```

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md (hunk 3 above)
Comment (wilsonwu) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md]: Added, will sync the zh version soon.

Comment (wilsonwu) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md:68]: All Done

**Comment by ericvn** [comment]
LGTM, but I can't approve until an @istio/wg-docs-maintainers-chinese does, as my approval would also cover that and I don't want the PR to merge without their approval.

**Review by windsonsea — APPROVED** [review]
LGTM for the zh part, thanks.

**Review by ericvn — APPROVED** [review]
Thanks.

**Pull request merged**